### ISSUE TITLE
Provide more relative Method

### ISSUE THREAD

**Issue by suryapratapsinghsuryavanshi** [suryapratapsinghsuryavanshi/solverjs#2]
With web application all the time we face many problems to implement some logic, but in logic, we want some common mathematical applications like GCD, sum of all digits and this makes our implementation more complex hence If you want all these common math problems solved, then you can use solverjs. SolverJS are a combination of many common mathematical and some very used logic solutions.

So we created many ways to solve the problem but you also added some more methods to make web dev easier.

**Comment by DevsNight** [comment]
I contributed. #3

**Comment by Rohit484** [comment]
JavaScript library

**Comment by suryapratapsinghsuryavanshi** [comment]
> I made a pull request #13 to optimize the getFib method to linear time rather than exponential.
> 
> There are other optimizations that can be made, let me know if I can go ahead and make pull requests for those as well after #13 is merged and confirmed.

Thanks, optimization is a major issue please contribute your way in further away as well.

**Comment by saranshsaini** [comment]
Ok sounds good. I just made another optimization that you can check out. I will look out for more.

**Comment by suryapratapsinghsuryavanshi** [comment]
Yes @saranshsaini, I have reviewed it, please consider my opinion and let me know.

**Comment by shyam-2002** [comment]
Hi, I have optimized the power function. Please go through my PR #22 .

**Comment by saranshsaini** [comment]
Hello, I made a quick fix #23 for reverseNumber so that it works properly for negative numbers. Let me know if you will merge it.

**Comment by suryapratapsinghsuryavanshi** [comment]
> Hello, I made a quick fix #23 for reverseNumber so that it works properly for negative numbers. Let me know if you will merge it.

Yeah, I merged it.
